orderbook/orderbook.c

```diff
@@ -289,10 +293,70 @@ PyMODINIT_FUNC PyInit_order_book(void)
         return NULL;
     }
 
+    st = get_order_book_state(m);
+
+    printf("allocating cached format\n");
```

Comment:
can we remove this printf?

Reply:
Sorry about that, completely missed it! Fixed now in amended commit.